source/sync/go-to-production/optimize-sync-atlas-usage.txt

```diff
@@ -19,11 +19,15 @@ Overview
 --------
 
 Atlas Device Sync uses space in your app's synced Atlas cluster to store utility
-data for sync. This includes a history of changes to each realm.
-Atlas App Services minimizes that space usage via backend
-compaction, which runs automatically. You can optimize further by
-specifying a client maximum offline time, which enables advanced backend
-compaction.
+data for sync. This includes a history of changes to each realm. 
+Atlas App Services minimizes this space usage via backend
```

Comment:
What are your thoughts on updating this to "minimizes the space usage in your Atlas cluster" and then removing the bullet points below since I dont think they offer too much